### ISSUE TITLE
[resize] keyboard control

### ISSUE THREAD

**Issue by travankor** [WayfireWM/wayfire#918]
The resize plugin should support keyboard-based resizing in addition to mouse-based resizing.

**Comment by miguelcaravantes** [comment]
i agree, it doesn't make sense to me enable the tile plugin and have to resize with mouse

**Comment by Vladimir-csp** [comment]
Openbox has a killer feature - move/resize to nearest border.
Also, a separate toggles for vertical and horizontal maximization.

**Comment by ammen99** [comment]
It should be rather easy these days to make an ipc script to resize views with a keybinding, and it can be customized to do it exactly as the user expects.
